gatling-http/src/test/scala/io/gatling/http/check/body/ConditionalCheckSpec.scala

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2011-2016 GatlingCorp (http://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.gatling.http.check.body
+
+import java.nio.charset.StandardCharsets._
+
+import scala.collection.mutable
+
+import io.gatling.{ ValidationValues, BaseSpec }
+import io.gatling.core.CoreDsl
+import io.gatling.core.check.CheckResult
+import io.gatling.core.config.GatlingConfiguration
+import io.gatling.core.session.Session
+import io.gatling.http.HttpDsl
+import io.gatling.http.response.{ StringResponseBody, Response }
+import io.gatling.commons.validation.Success
+import io.gatling.core.check.ConditionalCheck._

```

Comment:
I don't understand what is misplaced ?
You want to me to sort import in alphabetical order ? Is that it ?